packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -506,16 +507,16 @@ function TableRowGroup({children, ...otherProps}) {
   );
 }
 
-function TableRow({item, children, onAction, ...otherProps}) {
+function TableRow({item, children, ...otherProps}) {
   let ref = useRef();
   let state = useTableContext();
-  let allowsInteraction = state.selectionManager.selectionMode !== 'none' || onAction;
+  let {actions: {onRowAction}} = gridMap.get(state);
```

Comment:
better way to handle this one?

Reply:
I would just do what we had before, or maybe pass a `hasActions` prop from `TableView` to `TableRow` instead?